dissect/extfs/extfs.py

```diff
@@ -117,12 +117,12 @@ def journal(self) -> JDB2:
 
         return self._journal
 
-    def get(self, path: str | int, node: INode | None = None) -> INode:
-        if isinstance(path, int):
-            return self.get_inode(path)
+    def get(self, path_or_inum: str | int, node: INode | None = None) -> INode:
+        if isinstance(path_or_inum, int):
+            return self.get_inode(path_or_inum)
```

Comment:
Reviewer: [Possible existing issue]. This creates an inode without a parent node (old code)  / parent inum (new code). 

In case of a relative  link,  wouldn't this causes issues when resolving the target in `link_inode`?